build.xml

```diff
@@ -38,6 +38,16 @@
     <property name="scm.developerConnection" value="scm:git:ssh://git@github.com:datastax/cassandra.git"/>
     <property name="scm.url" value="scm:git:ssh://git@github.com:datastax/cassandra.git"/>
 
+    <!-- JDKs supported.
+        All releases are built with the default JDK.
+        Builds with non-default JDKs are considered experimental and for development and testing purposes.
+        When building, javac's source and target flags are set to the jdk used, so lower JDKs are not supported at runtime.
+        The use of both CASSANDRA_USE_JDK11 and use-jdk11 is deprecated.
+    -->
+    <property name="java.default" value="11" />
+    <!-- Not sure whether we need 17 below, at least jvm options were added some time ago for 17 -->
+    <property name="java.supported" value="1.8,11,17,22" />
```

Comment:
Maybe add 21?

Reply:
Is 1.8 actually still supported?  I don't think it is.  I would leave this as 11/17/22 as those are the JVMs we are using.

Reply:
Yeah, I think what we agreed is the cleaning of dead config for JDK8 is done in `main-5.0` but we can remove it from supported here.

Reply:
Given we are compiling our binaries with 11, I think we should start making things show meaningful errors if someone uses 8 rather than the strange error you get when it actually tries to run.

Reply:
Yes, I removed 1.8 and now it shows: `Unsupported JDK version used: 1.8`
Same as any other version which is not on that list of` java.supported`